robot/util/emailer/emailIPAddress.js

```diff
@@ -1,28 +1,52 @@
+const fs = require("fs").promises;
+const path = require("path");
+const ip = require('ip');
+const nodemailer = require('nodemailer');
 
+const RETRY_DELAY = 3000;
+const MAX_TRIES = 40;
+const EMAILS_FILE_NAME = "emails.txt";
+const DEFAULT_EMAILS = [
```

Comment:
Maybe add a comment here saying it's for in case there is not `emails.txt` file found?